packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.spec.ts

```diff
@@ -541,6 +541,24 @@ describe('RichTextEditor', () => {
             ).toBeTrue();
         });
 
+        it('should have only inner most nested list type button active', async () => {
```

Comment:
Is this new test covering anything that the existing test `should have "ul" tag name for bullet list and "ol" tag name for nested numbered list` didn't already cover? I don't notice anything. If that's the case then we can delete it.

Reply:
Missed to update the existing test cases before and added a new one. Will remove the newly added case.